czech.h

```diff
@@ -1,4 +1,51 @@
-#define SOUBOR FILE
-#define když if
-#define plav float
-#define číslo int
+#define SOUBOR      FILE
+
+#define nehybný     static
+#define brusle      inline
+#define měňavec     volatile
+#define autobus     auto
+#define venkovní    extern
+
+#define velikost    sizeof
+#define novytyp     typedef
+
+#define běž         goto
+#define pokud       if
+#define takne       else
+#define dělej       do
+#define dokud       while
+#define provse      for
+#define rozchoďit   continue
+#define rozhodit    break
+#define vrať        return
+#define přepínač    switch
+#define případ      case
+#define výchozí     default
+
+#define nabeton     true
+#define nepravda    false
+
+#define struktura   struct
+#define výčet       enum
+#define anarchie    union
+
+#define rozhodovák  bool
+#define neměnné     const
+#define podepsaný   signed
+#define bezcenný    unsigned
+#define nic         void
+#define cvalík      double
+#define plavec      float
+#define říman       int
+#define znk         char
+#define kraťas      short
+#define dlouhán     long
+#define hafodlouhej long long
+
+#define piš(...)    printf(__VA_ARGS__)
+#define fpiš(...)   fprintf(__VA_ARGS__)
+
+#define stdven      stdout
+#define stdchyb     stderr
+
```

Comment:
```suggestion
#define stddovnitr stdin
```
pro uplnost, kdyz uz tu mas stdout a stderr